app/tests/core_tests/test_grand_challenge_forge.py

```diff
@@ -65,6 +65,17 @@ def test_get_forge_json_description():
                 ]:
                     assert ci_key in component_interface
 
+    # Quick check on CI input and outputs
+    for index, ci in enumerate(
+        description["challenge"]["phases"][0]["inputs"]
```

Comment:
shouldn't it be algorithm_input here as well?

Reply:
It could? In this case, this refers to the JSON that is fed into the forge for package creation.

Since the pack is only generated for the algorithm-type phase, it seemed odd to call it `'algorithm_inputs'` in the pack/forge package itself.

Reply:
I think it would make it clearer, also because the evaluation method will have inputs and outputs.

Reply:
I now know why I did this: it was because of the predictions.json also mapping it to "inputs" and "outputs". Of, there it is clearly an algorithm job.

Reply:
I've updated will relying on CI to double-check if I replaced it correctly.